### ISSUE TITLE
Tree-Depth Plots 

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#305]
For random forest hyperparameter tuning. 

See [Finding Similar Days for Air Traffic Management](https://cloudfront.escholarship.org/dist/prd/content/qt0dn6d6h9/qt0dn6d6h9.pdf) for examples.

Create a parameter to set the y-axis to either concordance index or brier score:

http://scikit-learn.org/stable/modules/generated/sklearn.metrics.brier_score_loss.html
https://github.com/sebp/scikit-survival/blob/master/sksurv/metrics.py

**Comment by bbengfort** [comment]
Note: create a `yellowbrick.tree` package specifically for tree visualizers. See also #194

**Comment by bbengfort** [comment]
<img width="570" alt="screenshot 2018-03-21 23 26 03" src="https://user-images.githubusercontent.com/745966/37749670-6965b456-2d5f-11e8-8076-b0e222d4dfc8.png">
<img width="575" alt="screenshot 2018-03-21 23 26 15" src="https://user-images.githubusercontent.com/745966/37749671-69753822-2d5f-11e8-9f05-b4e8636324af.png">
<img width="596" alt="screenshot 2018-03-21 23 26 30" src="https://user-images.githubusercontent.com/745966/37749672-697f86ec-2d5f-11e8-80a5-2c96422cc656.png">
